lib/events.js

```diff
@@ -351,7 +351,7 @@ EventEmitter.prototype.prependOnceListener =
       return this;
     };
 
-// emits a 'removeListener' event iff the listener was removed
+// emits a 'removeListener' event if and only if the listener was removed
```

Comment:
Totally optional, but while you're in here, if you want, `emits` -> `Emits` and put a `.` at the end. I'm pretty sure we have Collaborators that strongly prefer that style (especially the capitalization), but I also know we have *lots* of comments that don't conform to it. So totally OK if you *don't* do it.